packages/material-ui-lab/src/Alert/Alert.js

```diff
@@ -0,0 +1,278 @@
+    border: `1px solid ${theme.palette.error.main}`,
+    '& $icon': {
+      color: theme.palette.error.main,
+    },
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="success"`. */
+  filledSuccess: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.success.main,
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="info"`. */
+  filledInfo: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.info.main,
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="warning"`. */
+  filledWarning: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.warning.main,
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="error"`. */
+  filledError: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.error.main,
+  },
+  /* Styles applied to the icon wrapper element. */
+  icon: {
+    marginRight: 12,
+    padding: '7px 0',
+    display: 'flex',
+    fontSize: 22,
+    opacity: 0.9,
+  },
+  /* Styles applied to the message wrapper element. */
+  message: {
+    padding: '8px 0',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+  },
+  /* Styles applied to the action wrapper element if `action` is provided. */
+  action: {
+    display: 'flex',
+    alignItems: 'center',
+    marginLeft: 'auto',
+    paddingLeft: 16,
+    marginRight: -8,
+  },
+});
+
+const defaultIconMapping = {
+  success: <SuccessOutlinedIcon fontSize="inherit" />,
+  warning: <ReportProblemOutlinedIcon fontSize="inherit" />,
+  error: <ErrorOutlineIcon fontSize="inherit" />,
+  info: <InfoOutlinedIcon fontSize="inherit" />,
+};
+
+const Alert = React.forwardRef(function Alert(props, ref) {
+  const {
+    action,
+    children,
+    classes,
+    className,
+    closeText = 'Close',
+    color,
+    icon,
+    iconMapping = defaultIconMapping,
+    onClose,
+    role = 'alert',
+    severity = 'success',
+    variant = 'text',
+    ...other
+  } = props;
+
+  return (
+    <Paper
+      role={role}
+      square
+      elevation={0}
+      className={clsx(
+        classes.root,
+        classes[`${variant}${capitalize(color || severity)}`],
+        className,
+      )}
+      ref={ref}
+      {...other}
+    >
+      {icon !== false ? (
+        <div className={classes.icon}>
+          {icon || iconMapping[severity] || defaultIconMapping[severity]}
+        </div>
+      ) : null}
+      <div className={classes.message}>{children}</div>
+      {action != null ? <div className={classes.action}>{action}</div> : null}
+      {action == null && onClose ? (
+        <div className={classes.action}>
+          <IconButton
+            size="small"
+            aria-label={closeText}
+            title={closeText}
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
If you want a tooltip use the tooltip component.

Reply:
It made me think of this change: https://github.com/mui-org/material-ui/pull/18283/files#r344488885.

Reply:
Same approach in https://github.com/mui-org/material-ui/blob/a07f18029a5532de310f234392c18664817f0dce/packages/material-ui/src/TablePagination/TablePagination.js#L156-L157

Reply:
The Tooltip component can be overkill for some use cases.

Reply:
The [tooltip guidelines](https://material.io/components/tooltips/tooltips.html) don't give any advice on using the browser tooltip. When should one be prioritised over the other?

Reply:
From my perspective, with an on-side example:

Tooltip pros:
- control the display delay, displays instantly,
- looks nice
![Capture d’écran 2020-01-02 à 00 18 46](https://user-images.githubusercontent.com/3165635/71647334-995f7100-2cf5-11ea-818a-0d3e44791fe2.png)

title pros:
- no bundle size impact
- displays after 2s, nice for non important labels.
![Capture d’écran 2020-01-02 à 00 19 43](https://user-images.githubusercontent.com/3165635/71647335-995f7100-2cf5-11ea-9a08-b6ba5e57cbd8.png)